### ISSUE TITLE
focus on one friend's updates (and maybe groups)

### ISSUE THREAD

**Issue by johlrogge** [agical/friendsunhosted#24]
One should be able to klick on a friend and see that friends activity. Right now the wazzup is an aggregate of all friends which is neat, it would be nice to complement it with seeing everything from one user/group. Similar to facebooks wall or twitters lists/ueser-stream

**Comment by jancborchardt** [comment]
Yeah, basically a profile page for every user, with all their posts.

**Comment by johlrogge** [comment]
exactly
